setup.py

```diff
@@ -36,7 +36,10 @@ def osx_setup():
     subprocess.run("sudo chmod 775 setup/macOS-setup.sh && setup/macOS-setup.sh",
                    shell=True, check=True)
 
-if platform.platform().lower().find('ubuntu') != -1 \
+if "microsoft" in platform.uname().release.lower():
```

Comment:
What if the computer is running Windows but does not have WSL (either because it hasn't been installed or enabled, or because it's running an older version of Windows that does not support WSL)? I would guess that it might still have `microsoft` in the platform name that Python sees.

Reply:
Hi @ragesoss, let me check this. Just a clarification, I'm I supposed to confirm that it doesn't run the debian script when the machine has only Python without WSL?

Reply:
I'm not sure. It should either work no matter what, or it should fail gracefully by detecting that the system isn't compatible with the scripts and exiting with an appropriate message.

Reply:
Okay then. In that case can I check if WSL is enabled or installed and run the debian script, else I can run the windows script? I included in the documentation that WSL is the recommended version but in case the version doesn't support WSL they can fallback to Windows script. Would this be okay?

Reply:
If you know that the debian script works on WSL, then that makes sense.